tests/fixtures/example_rule_missing_field.yml

```diff
@@ -0,0 +1,34 @@
+AnalysisType: rule 
+Filename: example_rule.py
+DisplayName: MFA Rule
+Description: MFA is a security best practice that adds an extra layer of protection for your AWS account logins.
+Severity: Critical
+Threshold: 5
+RuleID: AWS.CloudTrail.MFAEnabled
+Enabled: true
+SummaryAttributes:
+  - p_log_type
+  - p_any_ip_addresses
+LogTypes:
+  - AWS.CloudTrail
+Tags:
+  - AWS Managed Rules - Security, Identity & Compliance
+  - AWS
+  - CIS
+  - SOC2
+Runbook: >
+  Find out who disabled MFA on the account.
+Reference: https://www.link-to-info.io
+Tests:
+  -
+    Name: User MFA enabled passes compliance but fails dedup check.
+    ExpectedResult: true
+    Log:
+      {
+        "Arn": "arn:aws:iam::123456789012:user/test",
+        "CreateDate": "2019-01-01T00:00:00",
+        "CredentialReport": {
+          "MfaActive": true,
+          "PasswordEnabled": true
+        }
+      }
```

Comment:
You certainly can use JSON syntax here, but you don't have to. This would work as well:

```yml
    Log:
      Arn: arn:aws:iam::123456789012:user/test,
      CreateDate: '2019-01-01T00:00:00',
      CredentialReport:
        MfaActive: true,
        PasswordEnabled: true
```

Nitpick: be sure to add a newline at the end of the file as well (that will make the red dash in GitHub go away)